packages/core/strapi/lib/services/metrics/sender.js

```diff
@@ -44,13 +44,13 @@ module.exports = (strapi) => {
   const anonymousMetadata = {
     environment: strapi.config.environment,
     os: os.type(),
-    osPlatform: os.platform(),
+    platform: os.platform(),
```

Comment:
any reason this one is not osX like the other os properties ?

Reply:
@fdel-car I've mentioned it on Slack already: I'd prefer that too.

Reply:
As mentioned in my previous comment I arbitrarily kept `platform` because I thought it was unique enough to not be confused by anything else and I wanted feedback on that, so as you two have said the exact same thing I'll change that right way!